web/impact/impact/tests/test_mentor_program_office_hour_list_view.py

```diff
@@ -0,0 +1,164 @@
+# MIT License
+# Copyright (c) 2019 MassChallenge, Inc.
+
+import json
+from jsonschema import Draft4Validator
+
+from django.urls import reverse
+
+from accelerator.tests.factories import (
+    EntrepreneurFactory,
+    ExpertFactory,
+)
+
+from impact.tests.factories import MentorProgramOfficeHourFactory
+from impact.tests.api_test_case import APITestCase
+from impact.v1.helpers.mentor_program_office_hour_helper import (
+    OFFFICE_HOUR_FIELDS,
+)
+from impact.tests.utils import assert_fields
+from impact.v1.views import MentorProgramOfficeHourListView
+
+USER_COUNT = 3
+NON_USER_COUNT = 4
```

Comment:
from the names it's not clear what `USER_COUNT` and `NON_USER_COUNT`..and I'm not quite sure what they are either from the tests